tests/e2e/vc_service_util.go

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
Can we rename this to vc_utils.go as we can put other vc related operations also in this util such as enabling or checking some particular FSS on VC.